libs/wingsdk/src/cloud/bucket.ts

```diff
@@ -277,6 +277,36 @@ export interface IBucketClient {
    * @inflight
    */
   delete(key: string, opts?: BucketDeleteOptions): Promise<void>;
+
+  /**
+   * Tries to put an object in the bucket.
+   * @param key Key of the object.
+   * @param body Content of the object we want to store into the bucket.
+   * @returns `true` if the object was successfully stored, `false` if an error occurred.
+   */
+  tryPut(key: string, body: string): Promise<boolean>;
+
+  /**
+   * Tries to put a Json object in the bucket.
+   * @param key Key of the object.
+   * @param body Json object that we want to store into the bucket.
+   * @returns `true` if the object was successfully stored, `false` if an error occurred.
+   */
+  tryPutJson(key: string, body: Json): Promise<boolean>;
+
+  /**
+   * Tries to retrieve an object from the bucket.
+   * @param key Key of the object.
+   * @returns `true` if the object was successfully retrieved, `false` if an error occurred.
+   */
+  tryGet(key: string): Promise<boolean>;
```

Comment:
Please take a look at our SDK spec for bucket [here](https://docs.winglang.io/reference/wingsdk-spec#bucket). If the object exists, we still want to obtain its contents, so it should look something like this:
```suggestion
  /**
   * Get an object from the bucket if it exists
   * @param key Key of the object.
   * @returns the contents of the object as a string if it exists, nil otherwise
   */
  tryGet(key: string): Promise<string | undefined>;
```